code/game/objects/items.dm

```diff
@@ -171,7 +171,9 @@
 	var/sharpness = NONE
 
 	///How a tool acts when you use it on something, such as wirecutters cutting wires while multitools measure power
-	var/tool_behaviour = NONE
+	var/tool_behaviour = null
+	///Lazy list of all behaviours this tool can potentially adopt (but isn't necessarily active at the moment), must be manually maintained for now
+	var/list/all_tool_behaviours = null
```

Comment:
Mmm this might be better as a proc (ie, `get_all_tool_behaviors` than needing to instantiate a 1 length list on 80% of all items